src/main/cli/cli.c

```diff
@@ -4069,15 +4139,51 @@ static void cliSave(char *cmdline)
     cliReboot();
 }
 
+#if defined(USE_CUSTOM_DEFAULTS)
+static bool isDefaults(char *ptr)
+{
+    return strncmp(ptr, "# Betaflight", 12) == 0;
+}
+#endif
+
 static void cliDefaults(char *cmdline)
 {
-    bool saveConfigs;
+    bool saveConfigs = true;
+#if defined(USE_CUSTOM_DEFAULTS)
+    bool useCustomDefaults = true;
+#else
+    // Required to keep the linker from eliminating these
+    if (customDefaultsStart != customDefaultsEnd) {
+        delay(0);
+    }
+#endif
 
     if (isEmpty(cmdline)) {
-        saveConfigs = true;
     } else if (strncasecmp(cmdline, "nosave", 6) == 0) {
         saveConfigs = false;
+#if defined(USE_CUSTOM_DEFAULTS)
+    } else if (strncasecmp(cmdline, "bare", 4) == 0) {
+        useCustomDefaults = false;
+    } else if (strncasecmp(cmdline, "show", 4) == 0) {
+        char *ptr = customDefaultsStart;
+        if (isDefaults(ptr)) {
+            while (*ptr && ptr < customDefaultsEnd) {
```

Comment:
Need to handle 0xFF up here too. 
`while (*ptr && *ptr != 0xFF && ptr < customDefaultsEnd) {`